sections/main-product.liquid

```diff
@@ -359,6 +143,102 @@
                   {{ product.description }}
                 </div>
               {%- endif -%}
+              {%- when 'carousel' -%}
+                <cross-sell-carousel>
+                   <div class="glide" data-glide>
+                     <div class="glide__track" data-glide-el="track">
+                       <ul class="glide__slides glide__slides--cross-sell">
+                         {% for product in collections[block.settings.collection].products %}
+                           <li class="glide__slide">
+                               <div class="card card--cross-sell">
+                               <img src="{{ product.featured_image | img_url: '134x134' }}"  alt="{{ product.title }}"/>
+                               <p class="cross-sell-product-title">{{ product.title }}</p>
+                               <div {{ block.shopify_attributes }}>
+                               {%- if product != blank -%}
+                               <product-form class="product-form">
+                                 <div class="product-form__error-message-wrapper" role="alert" hidden>
+                                   <svg
```

Comment:
When doing shopify development, use snippets to render SVGs, it'll make the code easier to read.